### PR TITLE
Update privacy/security considerations

### PR DESCRIPTION
**Pull request by guidou**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-transform/pull/42.html" title="Last updated on Jun 15, 2021, 3:15 PM UTC (af98e28)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-transform/42/101978f...af98e28.html" title="Last updated on Jun 15, 2021, 3:15 PM UTC (af98e28)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -402,14 +402,33 @@ trackProcessor.readable
...
+{{MediaStreamAudioSourceNode}} and media elements. The security and privacy
+of {{MediaStreamTrackProcessor}} relies on the security and privacy of the
+{{MediaStreamTrack}} sources of the tracks to which {{MediaStreamTrackProcessor}}
+is connected. For example, camera, microphone and screen-capture tracks
+rely on explicit use authorization via permission dialogs (see
+[[MEDIACAPTURE-STREAMS]] and [[MEDIACAPTURE-SCREEN-SHARE]]),
+while element capture and {{MediaStreamTrackGenerator}}
+rely on the same-origin policy.
+A potential issue with {{MediaStreamTrackProcessor}} is resource exhaustion.
+For example, a site might hold on too many open {{VideoFrame}} objects
```
Comment (dogben) [inline index.bs:427]: typo "hold on *to* too many"

File: index.bs [hunk 2]
```diff
@@ -402,14 +402,33 @@ trackProcessor.readable
...
+and deplete a system-wide pool of GPU-memory-backed frames. UAs can
```
Comment (dogben) [inline index.bs:428]: Is it OK to abbreviate user agent?

**Review by dogben — APPROVED** [review]
LGTM. Sorry for slow followup, I didn't see this when you sent it.

Maybe also get review from @chcunningham, who [authored](https://github.com/w3c/webcodecs/commit/a92dc0249c2eec1af7bbf4e9ab98b6f5520e4bc3) the security and privacy section of the WebCodecs spec.

File: index.bs (hunk 2 above)
Comment (guidou) [reply index.bs:428]: Yes.

File: index.bs (hunk 1 above)
Comment (guidou) [reply index.bs:427]: Fixed

**Comment by guidou** [comment]
Merging. We can continue to improve this section in follow-up PRs, but this one is a net improvement as is.

**Pull request merged**